src/App/Controller/PublicDiscussionController.php

```diff
@@ -339,6 +426,8 @@ public function lockAction($slug)
         $em->persist($discussion);
         $em->flush();
 
+        $this->get('session')->getFlashBag()->add('info', 'The discussion has been locked');
```

Comment:
The application is in french but all flashbags are in english?

Reply:
Hmm yes for the moment :P We will deal with the translations later